ui/push-health/Health.jsx

```diff
@@ -60,7 +60,7 @@ export default class Health extends React.PureComponent {
     const expandedStates = Object.entries(metrics).reduce(
       (acc, [key, metric]) => ({
         ...acc,
-        [`${key}Expanded`]: metric.result !== 'pass',
+        [`${key}Expanded`]: metric.result === 'fail',
```

Comment:
Was causing the `parent` metric to be open by default because it is status of `none`.